.tav.yml

```
@@ -335,6 +335,9 @@ tedious:
   - versions: '16.0.0 || 16.1.0 || >16.1.0 <17' # first and last majors subset of '16.x'
     node: '>=16'
     commands: node test/instrumentation/modules/tedious.test.js
+  - versions: '>=17.0.0 <18' # first and last majors subset of '17.x' (as for now there is only v17.0.0)
+    node: '>=20'
```

```suggestion
    node: '18.x || >=20'
```

IIUC tedious@17 still supports Node v18: https://github.com/tediousjs/tedious/compare/v16.7.1...v17.0.0#diff-e98936aa52a6dd7416e4296e9628456227d834f7245967383fd9ff80fd985dadR49

---

yup. I misunderstood the commit message